flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -150,7 +153,7 @@ private static void testResult(
 
             assertEquals(
                     "Result of column [" + i + "] doesn't match.",
```

Comment:
`of spec` btw I think it would be nice to also print the test item summary to quickly know which one failed if there are many.

Reply:
Something like:
```
"Logical type for spec [" + i + "] of test [" + testItem + "] doesn't match"
&
"Result for spec [" + i + "] of test [" + testItem + "] doesn't match"
```

Reply:
Pushed this change, along with a fix regarding the relevant `toString()` implementation for TableAPI cases.